File: Основы программирования/1 курс/Лабораторная работа №2/Python/main.py
from os import system

x = float(input('Введіть координату x: '))
y = float(input('Введіть координату y: '))

if abs(x) <= 1:
    if (y <= 1):
        if (abs(x) <= y):
            ans = 'Точка належить заштрихованій області'
        else:
            ans = 'Точка не належить заштрихованій області'
    else:
        ans = 'Точка не належить заштрихованій області'
else:
    ans = 'Точка не належить заштрихованій області'

print(ans)

system('pause')

File: Основы программирования/1 курс/Лабораторная работа №2/C++/main.cpp
#include <iostream>

using namespace std;

/* Варіант 29
   Задані дійсні числа x, y. 
   Визначити, чи належить точка з координатами (x, y) 
   заштрихованій частині площини*/

int main()
{
	locale::global(locale("rus"));

	double x, y;
	string ans;

	cout << "Введiть координату x: ";
	cin >> x;
	cout << "Введiть координату y: ";
	cin >> y;

	if (abs(x) <= 1)
	{
		if (y <= 1)
		{
			if (abs(x) <= y)
			{
				ans = "Точка належить заштрихованiй областi";
			}
			else
			{
				ans = "Точка не належить заштрихованiй областi";
			}
		}
		else
		{
			ans = "Точка не належить заштрихованiй областi";
		}
	}
	else
	{
		ans = "Точка не належить заштрихованiй областi";
	}

	cout << ans << endl;

	system("pause");

	return 0;
}